paddle/gserver/layers/MKLDNNLRNLayer.cpp

```diff
@@ -29,7 +29,7 @@ bool MKLDNNLRNLayer::init(const LayerMap& layerMap,
   }
 
   /* the size of inputs for norm-layer is 1 */
-  CHECK_EQ(config_.inputs_size(), 1);
+  CHECK_EQ(config_.inputs_size(), 1UL);
```

Comment:
That will make compile error on some gcc version.

config_.inputs_size() is a integer, not a size_t.

Reply:
Thanks for @reyoung, but why there is a warning here?